extensions/3rdparty/tabber/tabber.js

```diff
@@ -550,7 +550,7 @@ function tabberAutomaticOnLoad(tabberArgs)
 /* Run tabberAutomaticOnload() unless the "manualStartup" option was specified */
 if (typeof tabberOptions == 'undefined') {
 
-	tabberAutomaticOnLoad();
+	$(tabberAutomaticOnLoad);
```

Comment:
What about `tabberAutomaticOnLoad(tabberOptions)` below? It should be wrapped the same way or move `$()` back to `tabberAutomaticOnLoad`

Reply:
Or is `$()` obsolete if we use `mw.hook`?